release_notes.md

```diff
@@ -4,15 +4,22 @@
 -->
 - Update Python Worker to 1.1.5 [Release Note](https://github.com/Azure/azure-functions-python-worker/releases/tag/1.1.5)
 - Update Python Library to 1.3.1 [Release Note](https://github.com/Azure/azure-functions-python-library/releases/tag/1.3.1)
-- **[BreakingChange]**[CustomHandler] If enableForwardingHttpRequest is false, http output binding response is expected to be a valid Json object with following optional fields :
-`{
-"statusCode" : "",
-"status" : "",
-"body": "",
-"headers" : {}
-}`
-Exception is thrown if HttpOutputBindingResponse is not valid Json.
-- **[BreakingChange]**[CustomHandler]Send query params as JObject and Identities as JArray PR #6621
+- **Breaking Changes in CustomHandler**
+    -  Issue #6644 : If enableForwardingHttpRequest is false, http output binding response is expected to be a valid Json object with following optional fields :
+    ```json
+    {
+    "statusCode" : "",
+    "status" : "",
+    "body": "",
```

Comment:
@anthonychu - Udpated release notes with breaking changes. We can include breaking change regarding deprecating `httpWorker` section when we make the code change.